### PR TITLE
Issue #67: Users should be able to run the tool in any directory

### PR DESCRIPTION
**Pull request by Luolc**
#67 

Maybe we need to add a test of running outside source directory in Travis after #15 is done.

### CODE REVIEW

**Comment by rnveach** [comment]
@Luolc #15 is done.
I don't feel running outside the directory needs to be added to CI. CI script must use specific directory as it is to compile package.
Please confirm locally that it now works outside source directory.

File: README.md [hunk 1]
```diff
@@ -93,6 +91,6 @@ $ java -jar target/regression-tool-XX-all.jar -r /path/to/checkstyle/ -p issue12
 
 ### Output
 
-The config file would be generated in `regression-tool` directory.
+The config file would be generated in current directory.
```
Comment (rnveach) [inline README.md]: `current` => `current working`

File: src/main/java/com/github/checkstyle/regression/extract/CheckstyleInjector.java [hunk 2]
```diff
@@ -139,14 +139,13 @@ private void invokeMavenCommand() throws InjectException {
      * @throws IOException failure of copying
      */
     private void copyInjectFilesToCheckstyleRepo() throws IOException {
-        final File srcDir =
-                new File("src/main/resources/com/github/checkstyle/regression/extract/");
+        final String path = "/com/github/checkstyle/regression/extract/";
         final File destDir = new File(repoPath, "src/test/java/com/puppycrawl/tools/checkstyle/");
         final String[] injections = {"ExtractInfoGeneratorTest.java", "JsonUtil.java"};
```
Comment (rnveach) [inline src/main/java/com/github/checkstyle/regression/extract/CheckstyleInjector.java]: We probably should have a test to verify all files in extract directory are listed in this string.
Please make a new issue.

File: README.md (hunk 1 above)
Comment (Luolc) [inline README.md]: Done.

File: src/main/java/com/github/checkstyle/regression/extract/CheckstyleInjector.java (hunk 2 above)
Comment (Luolc) [inline src/main/java/com/github/checkstyle/regression/extract/CheckstyleInjector.java]: #74

**Comment by Luolc** [comment]
@rnveach 
> Please confirm locally that it now works outside source directory

Tested on my local. Successful.

**Pull request merged**